allahcalc.mk

```diff
@@ -0,0 +1,26 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS += allahcalc
+ALLAHCALC_VERSION := 1.1
+DEB_ALLAHCALC_V ?= $(ALLAHCALC_VERSION)
+
+allahcalc-setup: setup
+	wget -q -nc -P (BUILD_SOURCE) https://github.com/clyde37/allah-calculator/releases/download/$(ALLAHCALC_VERSION)/allah_calculator_generator.cpp
```

Comment:
Use the ``GITHUB_ARCHIVE`` and ``EXTRACT_TAR`` functions, instead.

Reply:
I'm not sure why you would do that, but sure I can change it to work that way